test/katello_test_helper.rb

```diff
@@ -87,6 +89,7 @@ def setup_controller_defaults(is_api = false, load_engine_routes = true)
     set_default_locale
     setup_engine_routes if load_engine_routes
     @controller.stubs(:require_org).returns({})
+    load_permissions
```

Comment:
We have been seeing permissions loading errors in test runs. Let me recreate and I can elaborate more.